settings.gradle

```diff
@@ -1,10 +1,9 @@
 pluginManagement {
     repositories {
         gradlePluginPortal()
-        jcenter()
     }
     plugins {
-        id 'com.auth0.gradle.oss-library.java' version '0.14.0'
+        id 'com.auth0.gradle.oss-library.java' version '0.15.1'
```

Comment:
0.15.1 supports publishing to Maven Central directly, instead of Bintray, which is being retired.